index.js

```diff
@@ -56,8 +56,8 @@ program
   .option('-f, --force-reload', 'force a full reload of all site files when a file is changed')
   .option('-p, --port <port>', 'port for server to listen on (Default is 8080)')
   .option('-s, --site-config <file>', 'specify the site config file (default: site.json)')
-  .option('--one-page <file>', 'render and serve only a single page in the site')
-  .option('--no-open', 'do not automatically open the site in browser')
+  .option('-o, --one-page <file>', 'render and serve only a single page in the site')
+  .option('-n, --no-open', 'do not automatically open the site in browser')
```

Comment:
Since now they have shorthand commands, please do rearrange them so that everything lines up in alphabetical order. Thanks!